lib/check-dependencies.js

```diff
@@ -202,7 +202,36 @@ const checkDependenciesHelper = (syncOrAsync, config, callback) => {
             return;
         }
 
-        const depVersion = require(depJson).version;
+        const depJson = require(depJsonPath);
+
+        // Support NPM package aliases
```

Comment:
It's always lowercase:
```suggestion
        // Support npm package aliases
```